tests/url.test.ts

```diff
@@ -0,0 +1,62 @@
+import { assertStrictEquals as equals } from "../deps/assert.ts";
+import { getSite, testPage } from "./utils.ts";
+import urlPlugin from "../plugins/url.ts";
+
+Deno.test("url and htmlUrl update href", async () => {
+  const site = getSite({
+    dev: true,
+    src: "url",
+    location: new URL("https://example.com/test/"),
+  });
+
+  site.use(urlPlugin({}));
+
+  await site.build();
+
+  testPage(site, "/default-filter", (page) => {
+    equals(
+      page.document?.querySelector("#url")?.getAttribute("href"),
+      "https://example.com/test/url/"
+    );
+  });
+
+  testPage(site, "/default-filter", (page) => {
+    equals(
+      page.document?.querySelector("#htmlUrl")?.getAttribute("href"),
+      "https://example.com/test/htmlUrl/"
+    );
+  });
+
+});
+
+Deno.test("configure url and htmlUrl names", async () => {
+  const site = getSite({
+    dev: true,
+    src: "url",
+    location: new URL("https://example.com/"),
+  });
+
+  site.use(urlPlugin({
+    names: {
+      url: 'urlify',
+      htmlUrl: 'htmlUrlify',
+    },
+  }));
```

Comment:
Here you're using the url plugin twice, first is loaded by default and then you're reapply again here. After the suggested changes, this could be removed.

Reply:
Thanks Oscar. I will try to adjust it and send an update!